examples/openapi-ts-nuxt/package.json

```diff
@@ -7,19 +7,18 @@
     "build": "nuxt build",
     "dev": "nuxt dev",
     "generate": "nuxt generate",
-    "postinstall": "nuxt prepare",
+    "postinstall:enable-after-release": "nuxt prepare",
     "preview": "nuxt preview",
-    "typecheck": "tsc --noEmit"
+    "typecheck:old": "tsc --noEmit"
```

Comment:
This was typechecking the generated output which we don't need, so I disabled it for now. We'd still want to typecheck the source code.

Reply:
need to update the tsconfig.exclude probably